src/pages/login/login.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="ko">
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Login</title>
+  <link rel="stylesheet" href="/src/shared/font/font.css" />
+  <link rel="stylesheet" href="/src/shared/style/color.css" />
+  <link rel="stylesheet" href="./login.css" />
+  <title>판다마켓</title>
 </head>
 <body>
-  <div>LOGIN</div>
+<div class="flexBox">
+  <div class="mainContainer">
+    <div class="logoContainer">
+      <img src="/src/shared/img/logo/brand/logo.svg" alt="로고 이미지" class="logoImg"/>
+      <img src="/src/shared/img/logo/brand/logoText.svg" alt="로고 텍스트 이미지" class="logoTextImg"/>
+    </div>
+
+    <div class="inputContainer">
+      <label for="email">이메일</label>
+      <input id="email" placeholder="codeit@email.com"/>
+    </div>
+
+    <div class="inputContainer">
+      <label for="password">비밀번호</label>
+      <div class="passwordWrapper">
+        <input id="password" type="password" />
+        <button type="button" class="togglePassword">
+          <img src="/src/shared/img/button/btn_visibility_on.svg" alt="비밀번호 보기" />
+        </button>
+      </div>
+    </div>
+
+    <button class="loginButton">로그인</button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
        <button type="submit" class="sign-up-button">회원가입</button>
```

type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

```html
// case1: form에 onsubmit 이벤트 명시
<form class="login-form" onsubmit="login">
  <input /> // input에서 enter 키 입력시 onsubmit 이벤트 실행
  <button type="submit">login</button> // 해당 버튼 클릭 시 onsubmit 이벤트 실행
</form>

// case2: submit 버튼에 onclick 이벤트 명시
<form class="login-form">
  <input /> // input에서 enter 키 입력시 button의 onclick 이벤트 실행
  <button type="submit" onclick="login">login</button>
</form>
```

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)

Reply:
💊 제안
위처럼 동작할 수 있도록 form 태그로 관련 태그들을 감싸주세요!